test/test.py

```diff
@@ -596,8 +596,10 @@ def testTags(self):
             client.redis.flushdb()
             
             client.create_index((TextField('txt'), TagField('tags')))
+            
+            tags = 'foo,foo bar,hello;world'
 
-            client.add_document('doc1', txt = 'fooz barz', tags = 'foo,foo bar,hello;world')
+            client.add_document('doc1', txt = 'fooz barz', tags = tags)
```

Comment:
```
client.add_document('doc1', txt='fooz barz', tags=tags)
```

Reply:
added another document to ensure more complete testing